src/shared.ts

```diff
@@ -138,8 +138,19 @@ export const initStripeConnect = (
   stripePromise: Promise<StripeConnectWrapper>,
   initParams: IStripeConnectInitParams
 ): StripeConnectInstanceExtended => {
+  const eagerClientSecretPromise = (() => {
+    try {
+      return initParams.fetchClientSecret();
```

Comment:
Can we add an explicit nullcheck for `initParams`? Technically it should never be null, but some platforms don't use TS!
```suggestion
      if (!initParams?.fetchClientSecret) {
        throw new Error('fetchClientSecret should be defined');
      }
      return initParams.fetchClientSecret();
```

Reply:
If initParams is null, it would fail even without this change as we access `metaOptions` [later](https://github.com/stripe/connect-js/pull/122/files/d850c64fdcf4ff9788331200735132789c3bcf29#diff-3cc490302b79321eb38cc902ba2bc669600333b6e8428e4ac75981e3d5262550R224). However, could be helpful to do nullcheck and throw explicitly, will do that in a new PR as this is merged

Reply:
https://github.com/stripe/connect-js/pull/130